test/integration/patients/worklist/schedule.js

```diff
@@ -650,6 +656,20 @@ context('schedule page', function() {
 
         return fx;
       })
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.attributes.name = 'Test Action';
+
+        fx.data.relationships.flow = { data: { id: '1' } };
+
+        return fx;
+      })
+      .routePatientActions(_.identity, '2')
+      .routePatientFlows(_.identity, '2')
+      .routeFlow()
+      .routeFlowActions()
+      .routePatientByFlow()
```

Comment:
Ah I think `routePatientByAction()` may be needed as well for `cy.navigate('/patient/1/action/1');`